terminus-ui/chip/testing/src/test-components.ts

```diff
@@ -51,39 +54,34 @@ export class OneChip {
     <ts-chip
       [value]="option"
       [selected]="selected"
-      (removed)="removed()"
+      (remove)="removed()"
       (destroyed)="chipDestroy($event)"
       (selectionChange)="selectionChange($event)"
-    >
-    </ts-chip>
+    ></ts-chip>
   `,
 })
 export class SingleChip {
   public option = 'banana';
   public selected = false;
-  removed: (event?: TsChipEvent) => void = () => { }
-  chipDestroy: (event?: TsChipEvent) => void = () => { };
-  selectionChange: (event?: TsChipSelectionChange) => void = () => { };
+  public removed: (event?: TsChipEvent) => void = () => { };
```

Comment:
I'm suprised lint didn't add `;` here...

Reply:
Yeah, something is odd with our semicolon lint rules. Originally I couldn't get ESLint to fix that issue so I left TSLint in charge.. wondering if something is broken in this combined ES/TSLint stage..